Zhongli.Bot/Modules/Moderation/ModerationModule.cs

```diff
@@ -99,6 +99,22 @@ public async Task MuteAsync(IGuildUser user, TimeSpan? length = null, [Remainder
                 await ReplyReprimandAsync(result, details);
         }
 
+        [Command("say")]
+        [Summary("Make the bot send a message to the specified channel")]
+        [RequireAuthorization(AuthorizationScope.Helper)]
+        public async Task SayAsync(ITextChannel? channel, [Remainder] string message)
+        {
+            channel ??= (ITextChannel) Context.Channel;
+            await channel.SendMessageAsync(message);

```

Comment:
Probably better to have a setting or something here or at least set `AllowedMentions.None` for now so no accidental everyone ping.